### ISSUE TITLE
doesn't seem to work on onenote on windows 10

### ISSUE THREAD

**Issue by huxianglong** [rcmdnk/vim_ahk#37]
This is from the window spy:
![image](https://user-images.githubusercontent.com/23272422/69572859-0400ec80-0f93-11ea-9d03-893b99765de5.png)
How I can modify the settings to accommodate this?
windows version: 1809

**Comment by rcmdnk** [comment]
You can replace **OneNote** to **XXXX - OneNote**
or add **ahk_exe ApplicationFrameHost.exe**
in **Applictionas** of settings.

When I checked before, I thought the window name was always **OneNote**,
but it seems it has user name like **XXXX - OneNote**, depending on users.

Another problem is that these applications installed from Microsoft Store have the same ahk_class and ahk_exe

```
ahk_class ApplicationFrameWindow
ahk_exe ApplicationFrameHost.exe
```

If you set them in **Applictions** of vim_ahk, then vim_ahk will be enabled all of these applications.
It could be fine for some cases, though.

I will update README.

For now, I keep not to include **ahk_exe ApplicationFrameHost.exe** to avoid unexpected behaviors on **ahk_exe ApplicationFrameHost.exe** applications.
Maybe it can be changed in the future.

**Comment by huxianglong** [comment]
Yeah, all applications from window 10 share the same ahk_class, can we use
regex expressions to match the window title?

**Comment by rcmdnk** [comment]
There is an option in AHK:
[SetTitleMatchMode - Syntax & Usage | AutoHotkey v2](https://lexikos.github.io/v2/docs/commands/SetTitleMatchMode.htm)

I'll try to enable the option in vim_ahk.

**Comment by BlueDrink9** [comment]
I'm keen for regex-based matching too, since we could use it to enable this on certain web browser pages (eg google docs)

**Comment by rcmdnk** [comment]
This is now fixed in [[class][https://github.com/rcmdnk/vim_ahk/tree/class]] branch.

With the new branch, you have new settings of `SetTitleMatchMode`.
Default is 2.
If it is 2, the window title will be checked if any of VimGroup members are contained in the tile or not.
[SetTitleMatchMode - Syntax & Usage | AutoHotkey v2](https://lexikos.github.io/v2/docs/commands/SetTitleMatchMode.htm)

Therefore, `OneNote` enables vim_ahk on your OneNote.
This may also enable vim_ahk on such a browser if the browser opens a page with a title containing `OneNote`.

If you set `SetTitleMatchMode` to 3, it works as before (exact match).

I have still some ideas to update class branch but will merge the branch to master soon.

**Comment by rcmdnk** [comment]
Fixed at v0.6.0 (if SetTitleMatchMode is set to 2 (default))
